app/models/auth_sources/auth_source_ldap.rb

```diff
@@ -134,15 +134,14 @@ def update_usergroups(login)
                         .where(ExternalUsergroup.arel_table[:auth_source_id].eq(id))
                         .pluck(:id)
     # User's external user groups as of auth_source
-    usergroup_ids.concat(
-      ExternalUsergroup.where(auth_source_id: id)
-        .where(group_name_arel.in(external))
-        .pluck(:usergroup_id)
-    )
-
-    Usergroup.where(id: usergroup_ids.uniq).find_each do |usergroup|
-      refresh_usergroup_members(usergroup)
-    end
+    external_mapping_ids = ExternalUsergroup.where(auth_source_id: id)
```

Comment:
if we rename the variables to be more descriptive, this would perhaps be `updated_external_ids`?

Reply:
updated dosn't sound correct as those are ALL not just UPDATED ids :thinking:

Reply:
I went with `actual_external_ids` ... ?

Reply:
OR `correct_external_ids` ?